package/installation.suse-manager-server.xsl

```diff
@@ -415,7 +415,415 @@ textdomain="control"
                instead of omitting the key entirely -->
           <optional_default_patterns><![CDATA[]]></optional_default_patterns>
         </software>
-        </system_role>
+      </system_role>
+      <system_role>
+        <id>suma_multidisc_role</id>
+
+        <!-- the rest is overlaid over the feature sections and values. -->
```

Comment:
when I see this, maybe we should have something like inheritance of roles. So e.g. snapshots can be easily inherited or software selection.

Reply:
Long-term - I fully agree, since here quite some stuff is included 4-times. For SP2 ... we are a bit running out of time for that :-(

Reply:
Known issue caused by the way in which we currently replace sections of the control file in YaST (all or nothing).

We plan to fix it for next SP, if other priorities permit. But it's in the radar. See https://trello.com/c/QlQKxNbz/24-mechanism-to-refine-the-volumes-section-of-control-file